batch/Dockerfile

```diff
@@ -10,6 +10,11 @@ COPY batch/batch /batch/batch/
 RUN pip3 install --no-cache-dir /batch && \
   rm -rf /batch
 
+# https://cloud.google.com/storage/docs/gsutil/addlhelp/CRC32CandInstallingcrcmod
+RUN apt-get install gcc python-setuptools && \
```

Comment:
There's no apt cache when this command gets run. Also it should probably be `python3-setuptools`

Reply:
And `pip3` or `python3 -m pip`.

Reply:
apt-get is fixed, none of those pieces were required. `gsutil` prefers python2 to python3, so the choice of `python` is intentional.